docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -454,10 +581,84 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+
+Here is an example command created with the example set for `release-preprod`:
+
+```bash
+sudo iptables -A INPUT -s 192.168.1.75 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
 Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
 :::
+
+### Verify your signer is registered
```

Comment:
Can you add an examples of the information displayed when registered and not registered at the end of the section?

Reply:
**_Result in the explorer :_**

![image](https://github.com/input-output-hk/mithril/assets/135982616/947114a1-159a-4beb-8479-50f46ce94c39)

Is this ok for you ?

Reply:
Looks good!